js/src/popover.js

```diff
@@ -89,56 +84,22 @@ class Popover extends Tooltip {
     return this.getTitle() || this._getContent()
   }
 
-  getTipElement() {
-    if (this.tip) {
-      return this.tip
-    }
-
-    this.tip = super.getTipElement()
-
-    if (!this.getTitle()) {
-      SelectorEngine.findOne(SELECTOR_TITLE, this.tip).remove()
-    }
-
-    if (!this._getContent()) {
-      SelectorEngine.findOne(SELECTOR_CONTENT, this.tip).remove()
-    }
-
-    return this.tip
-  }
-
   setContent() {
     const tip = this.getTipElement()
 
     // we use append for html objects to maintain js events
```

Comment:
This comment should be moved to the definition of `_sanitizeAndSetContent` :thinking:

Reply:
but the  tip will not be shown as it checks for empty content first. Right?